core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
Yes it still works but the problem is that it isn't thread safe because we are running away from the use of libgdx safe threads. From their page "You application logic must be implemented in a ApplicationListener which has methods that get called by the Application when the application is created, resumed, paused, disposed or rendered." Game extends ApplicationListener.